layouts/shortcodes/pentest-report-requirements.html

```diff
@@ -1,3 +1,3 @@
-If you want a pentest report, you must set up a test of at least two credits. 
-If you have a one credit pentest, you'll still have access to the non-report
+If you want a pentest report, you generally must set up a test of at least two (2) credits. 
```

Comment:
We do support "occasional" reports based on one (1) 33-hour credit, under certain circumstances. I don't want to get into details, so I'm using vague language here.

We'll revise this again with DOCS-94.